mlflow/server/fastapi_app.py

```diff
@@ -7,7 +7,7 @@
 """
 
 from fastapi import FastAPI
-from starlette.middleware.wsgi import WSGIMiddleware
+from fastapi.middleware.wsgi import WSGIMiddleware
```

Comment:
ref: https://fastapi.tiangolo.com/advanced/wsgi/#including-wsgi-flask-django-others

Reply:
starlette is a dependency of fastAPI, though. Is it for not depending on transitive dependency?